build.gradle.kts

```diff
@@ -88,34 +82,43 @@ sourceSets.create("junitTest") {
 // -----------------------------------------------------------------------------
 // Tasks
 // -----------------------------------------------------------------------------
-
-
 tasks {
     //FIXME there is race condition with copying Version to generated and compiling project
     register<Copy>("templatesProcessing") {
-      val now = Date().time
-      val tokens = mapOf("project.version" to project.version, "build.timestamp" to "${now}")
-      inputs.properties(tokens)
-
-      from("src/main/java-templates") {
-        include("*.java")
-        filter<ReplaceTokens>("tokens" to tokens)
-      }
-      into("src/main/generated/io/knotx/launcher")
+        val now = Date().time
+        val tokens = mapOf("project.version" to project.version, "build.timestamp" to "${now}")
+        inputs.properties(tokens)
+
+        from("src/main/java-templates") {
+            include("*.java")
+            filter<ReplaceTokens>("tokens" to tokens)
+        }
+        into("src/main/generated/io/knotx/launcher")
```

Comment:
you are here have one spec, but you can have multiple so that generally  more strict is `from { into } ` // nested / paired